core/server/src/main/java/alluxio/master/file/FileSystemMaster.java

```diff
@@ -532,7 +532,8 @@ public PersistenceState getPersistenceState(long fileId) throws FileDoesNotExist
       Inode<?> inode = null;
       if (inodePath.fullPathExists()) {
         inode = inodePath.getInode();
-        if (inode.isDirectory() && ((InodeDirectory) inode).isDirectChildrenLoaded()) {
+        if ((inode.isDirectory() && ((InodeDirectory) inode).isDirectChildrenLoaded()) || inode
+            .isFile()) {
```

Comment:
Hi @peisun1115 . When you said that 1.1 changed the behavior, are you saying that in 1.0.1, `getFileInfo` would always call `ufs.exists()` and now it will only call it the first time?